metron-interface/metron-alerts/.nvmrc

```diff
@@ -0,0 +1 @@
+v9.11.1
```

Comment:
This version of node includes npm v5.6.0 which hasn't got `ci` command yet. Should `npm` be updated manually to the latest version? If yes, we should add the update command to the maven config as well.

Reply:
Thanks for pointing this out! The maven configuration is fine because we specify the npm version to install in the pom.xml (6.2.0). However, this will effect developers working on Metron. As far as I can tell, there isn't a way for us to include an npm version in our .nvmrc, so I think the next best thing we can do is add a step to the README that instructs developers to install npm 6.2.0.